index.js

```diff
@@ -1,9 +1,10 @@
+let fs = require("fs");
 let path = require("path");
 let { promisify } = require("faucet-pipeline-core/lib/util");
 let FileFinder = require("faucet-pipeline-core/lib/util/files/finder");
 
-let readFile = promisify(require("fs").readFile);
-let stat = promisify(require("fs").stat);
+let readFile = promisify(fs.readFile);
+let stat = promisify(fs.stat);
```

Comment:
https://nodejs.org/api/fs.html#fs_fs_promises_api suggests we might as well use `require("fs").promises`, because we no longer have to worry about ancient versions of Node?

Reply:
Yes. Will address this in my branch.